### PR TITLE
Add The Things Gateway connection guide

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This pull request is basically a do-over of #1368 with some additional changes.

#### Changes

- Based on #1368
- Adapted to new doc theme
- Cleanup

### CODE REVIEW

**Comment by rvolosatovs** [comment]
Blocking #1631

File: doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{{< figure src="ttkg-config-window.png" alt="TTKG Configuration Window" >}}
+
+Enter the following fields:
+
+1. **Name**: the **Gateway ID** that you chose earlier.
+2. Choose the WiFi network from the drop down and enter a password if necessary.
+
+Click the **Show Advanced Options** button and enter the following fields:
+
+1. **Account Server**: The URL of The Things Stack. If you're using a port other that `:443` then append that to the URL.
```
Comment (rvolosatovs) [inline doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md]: Is there some sane default we can suggest here?

Comment (htdvisser) [reply doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md]: Fixed the typo. Not sure what you mean with "sane default". I think people would know what to fill here if they don't have HTTPS on the usual port.

Comment (rvolosatovs) [reply doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md]: I meant the URL

Comment (htdvisser) [reply doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md]: Oh, okay, we could have used `https://thethings.example.com` as an example here

**Pull request merged**